migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -1,36 +1,29 @@
 [id="planning-migration-3-4"]
-= Planning your migration
+= Differences between {product-title} 3 and 4
 include::modules/common-attributes.adoc[]
 :context: planning-migration-3-4
 
 toc::[]
 
-Before performing your migration to {product-title} {product-version}, it is important to take the time to properly plan for the transition. {product-title} 4 introduces architectural changes and enhancements, so the procedures that you used to manage your {product-title} 3 cluster might not apply for {product-title} 4.
-
-[NOTE]
-====
-This planning document assumes that you are transitioning from {product-title} 3.11 to {product-title} {product-version}.
-====
+{product-title} {product-version} introduces architectural changes and enhancements/ The procedures that you used to manage your {product-title} 3 cluster might not apply to {product-title} 4.
```

Comment:
Typo that we should fix here:
s/enhancements/enhancements.

Reply:
Looks like that was fixed in another PR.

Reply:
Ah great! I think the reference I looked at for a translation question must have referenced and old instance. Thanks!